libs/blocks/preflight/panels/martech.js

```diff
@@ -21,8 +21,11 @@ function getTable(strings) {
 }
 
 async function checkMartechMeta() {
-  const strings = [...document.querySelectorAll('main :is(h1, h2, h3, h4, h5, h6, a)')]
-    .filter((el) => !el.closest('[class*="metadata"]') && !el.innerText.startsWith('http'))
+  const elementList = 'h1, h2, h3, h4, h5, h6, a, .tracking-header, .click-link, .con-button';
+  const allElements = `main :is(${elementList})`;
+  const strings = [...document.querySelectorAll(allElements)]
```

Comment:
nit: no need for `allElements` we can just do ``[...document.querySelectorAll(`main :is(${elementList})`)]...``

Reply:
@narcis-radu updated to not declare `allElements`